GuessTheMovie/ImageSearchAlgorithm.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <vector>
+#include <string>
+#include "Utils.h"
+#include "Link.h"
+#include "ImageSearcher.h"
+#include "Downloader.h"
+#include "SuggestionGenerator.h"
+
+
+struct ImageSearchAlgorithm
+{
+	SuggestionGenerator suggen;
+	ImageSearcher imsear;
+	Downloader down;
+	std::string folder_name;
+	void image_search_algorithm_in_use(const Film& film_name)
```

Comment:
I think there should either be a default constructor for `ImageSearchAlgorithm` for initializing `folder_name` or `image_search_algorithm_in_use ` should receive `folder_name` as an argument